frontend/website/src/Blog.re

```diff
@@ -26,6 +26,19 @@ let titleColor = c => Css.unsafe("--title-color", Style.Colors.string(c));
 let readMoreColor = c =>
   Css.unsafe("--read-more-color", Style.Colors.string(c));
 
+// Truncate HTML
+let rec truncate = (index, counter, html) =>
+  if (counter == 0) {
+    Js.String.substring(~from=0, html, ~to_=index + 4);
+  } else {
+    let indexOf = Js.String.indexOfFrom("</p>", index, html);
```

Comment:
When we looked at the generated html before, it looked like it was structured as a bunch of <p> tags wrapped in a <div>. If so, we should probably close the </div> after truncating. Can you confirm whether this is the case?

Reply:
just printed out the generated html: they're only `<p>` tags! we then set the inner HTML of a `div` tag to be the generated html

Reply:
Ah cool, thanks